FABulous/FABulous.py

```diff
@@ -523,6 +524,19 @@ def run_cmd(
 
 def main() -> None:
     """Entry point for the application."""
+    # Check if running on native Windows (not WSL)
+    if platform.system() == "Windows":
+        logger.error(
+            "FABulous is not officially supported on native Windows. "
+            "You are using it at your own risk - errors may occur.\n"
+            "For the best experience on Windows-based system, "
+            "please use Windows Subsystem for Linux (WSL).\n"
+            "For more information, visit: "
+            "https://docs.microsoft.com/en-us/windows/wsl/install"
+            "For all FABulous installation options, see:"
+            "https://fabulous.readthedocs.io/en/latest/getting_started/installation/index.html"
+        )
+
```

Comment:
I think we should exit directly if this is considered an error. Alternatively, we can prompt the user to confirm they really want to continue. You can use typer https://typer.tiangolo.com/tutorial/prompt/ 

This might be annoying for users who really decide to use Windows. We could consider adding a save for future, but I think this "save for future" can be done in another PR if there is real demand.